### PR TITLE
Prac 07 feedback

### PR DESCRIPTION
**Pull request by JarrodE**
Hey mate @Jackcox343 can you please review my code! Mainly in particular my docstrings and commits are properly done. Cheers!

### CODE REVIEW

File: prac_07/project_management.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 
 def main():
+    """Main function to run the project management program."""
```
Comment (jackcox343) [inline prac_07/project_management.py:22]: Nice and clear

File: prac_07/project_management.py [hunk 2]
```diff
@@ -41,6 +42,7 @@ def main():
 
 
 def load_projects(filename):
+    """Load the projects from a given filename and return them as a list of Project objects."""
```
Comment (jackcox343) [inline prac_07/project_management.py:46]: nice and clear!

File: prac_07/project_management.py [hunk 3]
```diff
@@ -56,6 +58,7 @@ def load_projects(filename):
 
 
 def save_projects(filename, projects):
+    """Saves projects to a given file."""
```
Comment (jackcox343) [inline prac_07/project_management.py:62]: Could this give information of formatting

File: prac_07/project_management.py [hunk 4]
```diff
@@ -64,6 +67,7 @@ def save_projects(filename, projects):
 
 
 def display_projects(projects):
+    """Display the projects, separating incomplete and complete projects."""
```
Comment (jackcox343) [inline prac_07/project_management.py:71]: nice and clear

File: prac_07/project_management.py [hunk 5]
```diff
@@ -83,6 +88,7 @@ def filter_projects(projects):
 
 
 def add_project(projects):
+    """Add a new project."""
```
Comment (jackcox343) [inline prac_07/project_management.py:92]: Mabye "This function adds a new project"?

File: prac_07/project_management.py [hunk 6]
```diff
@@ -93,6 +99,7 @@ def add_project(projects):
 
 
 def update_project(projects):
+    """Update a project."""
```
Comment (jackcox343) [inline prac_07/project_management.py:103]: Mabye "This function updates a project"?

File: prac_07/project.py [hunk 7]
```diff
@@ -6,20 +6,25 @@
 
 
 class Project:
+    """Represent a project object."""
```
Comment (jackcox343) [inline prac_07/project.py]: Nice and Clear

Comment (JarrodE) [reply prac_07/project.py]: Thanks mate!

**Pull request merged**